assets/js/admin/settings.js

```diff
@@ -14,6 +14,8 @@
 			boxWidth: '400px',
 			columnClass: 'evf-responsive-class'
 		};
+
+		$('.everest-forms .everest_forms_recaptcha_type:checked').trigger("change");
```

Comment:
Explain to me the scenario of this use case here?

Reply:
We have to trigger change event here because on page load we only have to show the site and secret key only for selected reCaptcha type.